phongo_compat.h

```diff
@@ -92,9 +92,17 @@
 #  define ARG_UNUSED
 #endif
 
-#if SIZEOF_LONG == 4
+#if PHP_VERSION_ID < 70000
+typedef long zend_long;
+# define SIZEOF_ZEND_LONG SIZEOF_LONG
```

Comment:
The intention here is to allow use of `zend_long` for `zend_parse_parameters()` and simplify our architecture checks later, where we might otherwise have to check `SIZEOF_LONG` and `SIZEOF_ZEND_LONG` for PHP 5 and 7, respectively.